staging/src/k8s.io/code-generator/generate-groups.sh

```diff
@@ -52,6 +52,8 @@ shift 4
   cd "$(dirname "${0}")"
   go install ./cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
 )
+# We expect the above commands to get installed here:
+gobin="$(go env GOBIN)" 
```

Comment:
why not GOBIN?

Reply:
I prefer lower-case variable names when they are not exported.
But I can change to upper case if you feel strongly about this..

Reply:
Can you PTAL @hwdef ? That part changed somewhat since you reviewed.

Reply:
I think both of lower-case and higher-case are good, 
I do not have approve authority, but I can add a LGTM label for this PR.